deploy-board/deploy_board/templates/groups/group_config.tmpl

```diff
@@ -339,5 +364,8 @@
         $("#targetgroupsExpressionId").click(function() {
             $("#targetgroupsExpressionDetailsId").collapse('toggle');
         });
+        $("#runbooklinkExpressionId").click(function() {
+            $("#runbooklinkExpressionDetailsId").collapse('toggle');
+        });
```

Comment:
non-blocking. This and all above can share the same class and this only need to apply to the class once.